### PR TITLE
Fix: Organization cards on landing page visual bug

### PR DESCRIPTION
**Pull request by JoshKornfeld**
## Description

Solves #1026 

- If an organization has more than 1 type, the chips will stack on top of each other rather than be next to each other
- The summary text now goes into the next line and will cut off like it does in /browse#organizations

## Test plan

- Visit the landing page
- Scroll down to "Find a climate action organization and get involved"
- The cards should be visible there

## Before landing

1. PR has meaningful title
1. `yarn lint` passes (frontend)
1. `yarn format` passes (frontend)
1. `make format` passes (backend)

### CODE REVIEW

File: frontend/src/components/organization/OrganizationPreview.js [hunk 1]
```diff
@@ -21,14 +21,15 @@ const useStyles = makeStyles((theme) => {
       "-moz-user-select": "none",
       "-ms-user-select": "none",
       userSelect: "none",
-      backgroundColor: "#f7f7f7",
+      backgroundColor: "#f7f7f7", // f7f7f7
```
Comment (JoshKornfeld) [inline frontend/src/components/organization/OrganizationPreview.js:24]:
```suggestion
      backgroundColor: "#f7f7f7"
```

Comment (JoshKornfeld) [reply frontend/src/components/organization/OrganizationPreview.js:24]: Forgot to delete a comment while testing things

File: frontend/src/components/organization/OrganizationPreviewHeader.js [hunk 2]
```diff
@@ -13,9 +13,10 @@ const useStyles = makeStyles((theme) => {
       textOverflow: "ellipsis",
       display: "inline-block",
     },
-    headerWrapper: {
+    headerWrapper: (props) => ({
       justifyContent: "center",
-    },
+      marginTop: props.numOfTypes > 1 ? 0 : 21,
```
Comment (ddhanesha) [inline frontend/src/components/organization/OrganizationPreviewHeader.js:18]: What is 21 here? How did you define that number?

File: frontend/src/components/organization/OrganizationPreviewHeader.js [hunk 3]
```diff
@@ -26,16 +27,21 @@ const useStyles = makeStyles((theme) => {
...
+      alignItems: "center",
       marginTop: "-15px",
     },
   };
 });
 
 export default function OrganizationPreviewHeader({ organization }) {
-  const classes = useStyles();
+  const classes = useStyles({
+    numOfTypes: organization.types.length,
```
Comment (ddhanesha) [inline frontend/src/components/organization/OrganizationPreviewHeader.js:43]: I think better variable name would be `orgTypeLength`.

**Review by ddhanesha — CHANGES_REQUESTED** [review]

**Review by ddhanesha — APPROVED** [review]
Let's get this merged!

**Pull request merged**